quickjs.c

```diff
@@ -37,6 +37,9 @@
 #include <timezoneapi.h>
 #endif
 #endif
+#if defined(_MSC_VER)
+#include <intrin.h>
```

Comment:
I see it included in line 46, but in a not MSVC block, can you rearrange that instead?

Reply:
Now just in an `#if defined(_WIN32)` block, so it's always included when targeting Windows.